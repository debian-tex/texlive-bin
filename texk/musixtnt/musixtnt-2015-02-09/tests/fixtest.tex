\input musixtex.tex
\input musixsty.tex
\input musixps
\input musixtnt
\nohalfties
\def\pstiehgt{0.5}
\def\psslurhgt{0.5}
\vsize9.75in
\voffset-0.5in
\hsize7.0in
\hoffset-0.25in
\smallmusicsize
\bigaccid
\advance\parindent by 10pt
\def\raisebarno{5\internote}%
\nobarnumbers
\relativeaccid
\geometricskipscale
\instrumentnumber1
\setclef1\bass\setbassclefsymbol1\bassoct
\setclef4\treble\settrebleclefsymbol4\trebleoct
\generalsignature{+1}
\generalmeter\meterC
\setname1{B}
\setname2{T}
\setname3{A}
\setname4{S}
\fulltitle{Quodlibet}
\subtitle{Variation 30 of the Goldberg Variations for Clavier}
\title{Quodlibet}
\author{J. S. Bach (1685--1750)}
\shortauthor{J. S. Bach}
\othermention{Arranged for Recorders by R. D. Tennent}
\maketitle
\startbarno=0
\nobarnumbers
\startpiece\addspace\afterruleskip%
\TransformNotes{#2&#3&#4&#5}{#2}%
\systemnumbers%
\Notes\ds&\cu d&\ds&\ds\en
\bar%1
\Notes\hl N&\ibu1g2\qb1{ghi}\tqh1j&&\hpause\en
\Notes\hl M&\ibl1k{-2}\qb1k&&\en
\notes&\nbbl1\qb1j\tqb1i&&\qp\en
\Notes&\ibu1h0\qb1h&&\ds\en
\notes&\nbbu1\qb1g\tqh1h&&\cu d\en
\def\atnextbar{\znotes&&\centerpause&\en}%
\bar%2
\Notes\hl L&\ibu1i{-2}\qb1i\tqh1h&\ql n&\ibu3g2\qb3{gh}\en
\Notes&\itied1g\qu g&\ql n&\qb3i\tqh3j\en
\Notes\ql{.K}&\ttie1\qu g&\ql o&\ibl3k{-2}\qb3k\en
\notes&&&\nbbl3\qb3j\tqb3i\en
\Notes&\ibu1f0\shake{n}\qb1f&\cl o&\cu h\en
\notes\cu J&\nbbu1\qb1e\tqh1f&\ds&\ds\en
\bar%3
\Notes\hu I&\qu g&\ibl2n{-1}\qb2{no}&\ql k\en
\Notes&\ds\cl k&\qb2{n}\tqb2{=m}&\ql k\en
\Notes\hu J&\ibu1j{-2}\qb1{ji}&\ibl2j{0}\qb2{lk}&\ql l\en
\Notes&\qb1h\tqh1g&\qb2j\tqb2p&\ql l\en
\bar%4
\Notes\hl K&\itied1f\shake{n}\qu f&\ibl2o{-1}\qb2{on}&\ibl3k{-1}\qb3{kl}\en
\notes&\ttie1\ibbu1e{-2}\qb1f\tbbu1\qb1e&\qb2o&\qb3k\en
%\vnotes1.5\elemskip&\nbbbu1\qb1e\tbbu1\qb1f&&\en
\Notes&\itied1d\tqh1d&\tqb2m&\tqb3j\en
\Notes\qu G&\ttie1\qu d&\ibl2n{-2}\qb2n\tqb2m&\ibu3h{-1}\shake{p}\qb3i\tqh3h\en
\Notes\ds&\qp&\ibl2n{-5}\qb2n&\qu g\en
\notes\ibbl0b{-1}\qb0b\tqb0a&&\tqb2k&\en
\bar%5
\Notes\ibl0M0\qb0N&\ql n&\ibl2n2\qb2n&\hpause\en
\notes\nbbl0\qb0M\tqb0N&&\qb2o&\en
\Notes\ibl0K0\qb0L&\ql n&\qb2p&\en
\notes\nbbl0\qb0K\tqb0L&&\tqb2q&\en
\Notes\ql{.M}&\ql o&\ibl2r{-2}\qb2r&\qp\en
\notes&&\nbbl2\qb2q\tqb2p&\en
\Notes\sk\cu I&\ql o&\ibl2o6\qb2o\itieu2r\tqb2r&\ds\cl k\en
\bar%6
\Notes\ql{.L}&\ibl1n{-1}\qb1{no}&\ttie2\ibl2r{-2}\qb2r\tqb2q&\ibl3n{-2}\qb3{nm}\en
\Notes&\qb1n&\ql p&\qb3l\en
\notes\ibbl0M2\qb0M\tqb0N&\tqb1m&&\tqb3k\en
\Notes\itieu0a\ql a&\ibl1l{-2}\qb1l&\ql{.o}&\shake{n}\ibl3i0\qb3{^j}\en
\notes&\tqb1k&&\nbbl3\qb3i\tqb3j\en
\notes\ttie0\ibbl0a{-2}\qb0{aN}&\cl{^j}&&\cu h\en
\notes\qb0M\tqb0L&\ds&\ibbl2i2\qb2i\qsk\tqb2{^j}&\ds\en
\bar%7
\Notes\ql M&\ibu1g0\qb1{fg}&\ibl2k0\qb2{kl}&\qu h\en
\Notes\ql K&\qb1h\tqh1f&\qb2m\tqb2k&\qu h\en
\Notes\ql N&\ibu1g{-2}\qb1{gf}&\ibl2i4\qb2{ik}&\ql i\en
\Notes\qu G&\qb1e\tqh1d&\qb2n\tqb2m&\ql i\en
\bar%8
\Notes\qu{.H}&\ibu1c4\qb1{^cd}&\ibl2l{-1}\qb2{.l}\roff{\itieu2k\tbbl2\tqb2k}\qsk&\ibu3i{-1}\qb3{hi}\en
\notes&\qb1e&\ttie2\ibbl2j0\qb2k\hqsk\tbbl2\qb2{^j}&\qb3h\en
\Notes\cu{^J}&\itied1h\tqh1h&\itieu2l\tqb2l&\tqh3g\en
\notes\ql K&\ttie1\ibu1h{-2}\qb1h&\ttie2\ibbl2l{-1}\qb2{lk}&\ibu3f{-1}\qb3f\en
\notes&\tqh1g&\qb2l\tqb2j&\tqh3e\en
\Notes\cl K&\cu f&\cl k&\cu d\en
\advance\barno by -1%
\leftrightrepeat
\Notes\ds&\cl m&\ds&\cl o\en
\bar%9
\Notes\ql d\sk\ql c&\ibl1k0\qb1{kmk}\tqb1{m}&\ql o\sk\ql o&\ibl3m0\qb3{mom}\tqb3o\en
\Notes\ql b&\ibl1k0\qb1n&\ql p&\ibl3k2\qb3k\en
\notes&\qb1k&&\nbbl3\qb3l\tqb3m\en
\Notes\ql N&\qb1k\tqb1n&\ql p&\ibl3n{-4}\qb3n\tqb3l\en
\bar%10
\Notes\ibl0b{-1}\qb0{cbc}\tqb0a&\cl l\ds\qp&\ibl2o{-1}\qb2{opo}\tqb2n&\ibl3o{-2}\qb3{onm}\tqb3l\en
\Notes\ql{.b}&\ql m&\ibl2m{-2}\qb2m&\ibl3j0\shake n\qb3{^k}\en
\notes&&\tqb2l&\nbbl3\qb3{^j}\tqb3k\en
\Notes\sk\cl a&\ql m&\cl{^k}\ds&\ibl3i8\qb3i\tqb3{p}\en
\bar%11
\Notes\ql{N}\sk\ql{N}&\ibl1l0\qb1{nln}\tqb1l&\ql p\sk\ql p&\ibl3m0\qb3{lpl}\tqb3{p}\en
\notes\ql{.a}&\ibl1m{-1}\qb1{m*n}&\ql q&\ibbl3k2\qb3{^klm}\tqb3n\en
\Notes&\qb1m&\ql q&\ibl3o{-2}\qb3o\en
\notes\cl{^a}&\tqb1l&&\nbbl3\qb3n\qsk\tqb3{m}\en
\bar%12
\Notes\ql b&\ibl1k2\qb1{^k}&\ibl2o{-1}\qb2p&\ibl3n{-2}\qb3n\en
\notes&\nbbl1\qb1l\tqb1m&\qb2q&\nbbl3\qb3m\tqb3l\qsk\en
\notes\qu I&\itied1i\qu i&\qb2p&\ibbl3k0\qb3{^kl}\en
\notes&&\tqb2o&\qb3m\tqb3k\en
\Notes\ql{.L}&\ttie1\ibu1i0\qb1i&\ibl2n0\qb2n&\itieu3l\hl l\en
\notes&\nbbu1\qb1h\tqh1i&\nbbl2\qb2m\tqb2n&\en
\Notes&\ibu1g2\qb1g&\cl l&\en
\notes\cl{=K}&\nbbu1\qb1h\tqh1i&\ds&\en
\bar%13
\notes\qu J&\ibl1j2\qb1j\sk\qb1k&\hpause&\ttie3\ibbl3k1\slur lku1\qb3l\qb3k\slur mlu1\qb3m\tqb3l\en
\notes\ql c&\qb1l\sk\tqb1m&&\ibbl3m1\isluru3n\qb3n\tbbl3\tslur3m\qb3m\tqb3o\en
\notes\ql{.b}&\ibl1n{-1}\qb1n\sk\nbbl1\qb1m\tqb1l&\ql n&\ql{.k}\en
\Notes&\cl k&\ibl2i{-5}\qb2n&\en
\notes\ibbu0J{-2}\qb0J\tqh0I&\ds&\tqb2g&\ibbl3j2\qb3j\tqb3k\en
\bar%14
\Notes\qu{.H}&\ds&\ibl2j1\qb2{.j}&\ibl3l2\qb3l\en
\notes&\cu h&\sk\tbbl2\tqb2k&\nbbl3\qb3m\tqb3n\en
\Notes&\ibu1e0\qb1e&\ibl2j{-2}\qb2j&\ibl3o{-2}\qb3o\en
\notes\ibbu0I2\qb0I\tqh0J&\nbbu1\qb1d\tqh1e&\tqb2{i}&\tqb3n\en
\notes\ql{.K}&\ibu1g1\qb1f&\ibu2h1\qb2h&\ibbl3m{-2}\shake{o}\qb3m\en
\notes&&&\tbbl3\qb3l\en
\Notes&\itied1g\tqh1g&\tqh2i&\tqb3k\en
\Notes&\ttie1\ibu1g{-1}\qb1g&\ibl2j2\qb2j&\ibu3i{-6}\qb3j\en
\notes\ibbu0J2\qb0J\tqh0K&\tqh1f&\tqb2k&\tqh3d\en
\bar%15
\Notes\ibu0I0\qb0G&\qu d&\ibu2i0\qb2i&\ibl3i0\qb3i\en
\notes\qb0H&&\nbbu2\qb2h\tqh2i&\qb3j\en
\Notes\qb0I&\qu d&\cu g&\qb3k\en
\Notes\tqh0G&&\ds&\tqb3i\en
\Notes\qu{.J}&\qu e&\ds&\ibu3j{-2}\qb3j\en
\notes&&\ibbl2l{-2}\qb2l\tqb2k&\qb3i\en
\Notes&\qu e&\ibl2j4\qb2j&\qb3h\en
\Notes\cu{^J}&&\tqb2l&\tqh3g\en
\bar%16
\notes\ql K&\ibu1g{1}\qb1d&\ibbl2h1\qb2{hk}&\ibu3f2\qb3f\en
\notes&\qb1e&\qb2j\tqb2i&\tqh3g\en
\notes\ql K&\qb1d&\ibbl2j0\qb2j\tbbl2\qb2h&\ibu3h{-4}\qb3h\en
\Notes&\tqh1j&\itieu2k\tqb2k&\tqh3f\en
\Notes\ql N&\ibu1i{-2}\qb1i&\ttie2\ibl2j0\qb2k&\qu g\en
\notes&\tqh1h&\nbbl2\qb2j\tqb2k&\en
\Notes\cu G&\cu g&\cl i&\cu g\en
\setrightrepeat
\endpiece%
\eject
\end
