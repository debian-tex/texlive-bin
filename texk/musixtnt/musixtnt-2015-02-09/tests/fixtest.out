\input musixtex.tex
\input musixsty.tex
\input musixps
\input musixtnt
\nohalfties
\def\pstiehgt{0.5}
\def\psslurhgt{0.5}
\vsize9.75in
\voffset-0.5in
\hsize7.0in
\hoffset-0.25in
\smallmusicsize
\bigaccid
\advance\parindent by 10pt
\def\raisebarno{5\internote}%
\nobarnumbers
\relativeaccid
\geometricskipscale
\instrumentnumber1

\setclef1\bass\setbassclefsymbol1\bassoct
\setclef4\treble\settrebleclefsymbol4\trebleoct
\generalsignature{+1}
\generalmeter\meterC
\setname1{B}
\setname2{T}
\setname3{A}
\setname4{S}
\fulltitle{Quodlibet}
\subtitle{Variation 30 of the Goldberg Variations for Clavier}
\title{Quodlibet}
\author{J. S. Bach (1685--1750)}
\shortauthor{J. S. Bach}
\othermention{Arranged for Recorders by R. D. Tennent}
\maketitle
\startbarno=0
\nobarnumbers
\setstaffs11
\nostartrule
\startpiece\addspace\afterruleskip%
\TransformNotes{#2}{#2}%
\systemnumbers%
\Notes\ds\en
\bar%1
\NOTes\hl N\en
\NOTes\hl M\en
\bar%2
\NOTes\hl L\en
\NOtes\ql{.K}\en
\Notes\cu J\en
\bar%3
\NOTes\hu I\en
\NOTes\hu J\en
\bar%4
\NOTes\hl K\en
%\vnotes1.5\elemskip&\nbbbu1\qb1e\tbbu1\qb1f&&\en
\NOtes\qu G\en
\Notes\ds\en
\notes\ibbl0b{-1}\qb0b\tqb0a\en
\bar%5
\Notes\ibl0M0\qb0N\en
\notes\nbbl0\qb0M\tqb0N\en
\Notes\ibl0K0\qb0L\en
\notes\nbbl0\qb0K\tqb0L\en
\NOtes\ql{.M}\en
\Notes\cu I\en
\bar%6
\NOtes\ql{.L}\en
\notes\ibbl0M2\qb0M\tqb0N\en
\NOtes\itieu0a\ql a\en
\notes\ttie0\ibbl0a{-2}\qb0{aN}\en
\notes\qb0M\tqb0L\en
\bar%7
\NOtes\ql M\en
\NOtes\ql K\en
\NOtes\ql N\en
\NOtes\qu G\en
\bar%8
\NOtes\qu{.H}\en
\Notes\cu{^J}\en
\NOtes\ql K\en
\Notes\cl K\en
\advance\barno by -1%
\leftrightrepeat
\Notes\ds\en
\bar%9
\NOtes\ql d\ql c\en
\NOtes\ql b\en
\NOtes\ql N\en
\bar%10
\Notes\ibl0b{-1}\qb0{cbc}\tqb0a\en
\NOtes\ql{.b}\en
\Notes\cl a\en
\bar%11
\NOtes\ql{N}\ql{N}\en
\NOtes\ql{.a}\en
\Notes\cl{^a}\en
\bar%12
\NOtes\ql b\en
\NOtes\qu I\en
\NOtes\ql{.L}\en
\Notes\cl{=K}\en
\bar%13
\NOtes\qu J\en
\NOtes\ql c\en
\NOtes\ql{.b}\en
\notes\ibbu0J{-2}\qb0J\tqh0I\en
\bar%14
\NOtes\qu{.H}\en
\notes\ibbu0I2\qb0I\tqh0J\en
\NOtes\ql{.K}\en
\notes\ibbu0J2\qb0J\tqh0K\en
\bar%15
\Notes\ibu0I0\qb0G\en
\Notes\qb0H\en
\Notes\qb0I\en
\Notes\tqh0G\en
\NOtes\qu{.J}\en
\Notes\cu{^J}\en
\bar%16
\NOtes\ql K\en
\NOtes\ql K\en
\NOtes\ql N\en
\Notes\cu G\en
\setrightrepeat
\endpiece%
%\eject

\end
